README.md

```diff
@@ -8,6 +21,53 @@ high fantasy theme, featuring both singleplayer and online/hotseat multiplayer
 combat. Fight a desperate battle to reclaim the throne of Wesnoth, or take
 hand in any number of other adventures.
 
+Contributing
+-------------
+Interested in creating your content? One of Wesnoth's best features is its extensibility. Players can create new maps, units, races, scenarios, art, music, and even entire campaigns! Please read information on our site (https://wiki.wesnoth.org/Create) for more information.
+
+**What can players contribute?!**
+
+ -WML Tutorial - Introduction to WML (In Progress)
```

Comment:
Introduction -> introduction, for consistency with the rest of the list below it.